client/storage/src/overrides/mod.rs

```diff
@@ -124,7 +124,7 @@ where
 
 	pub fn account_storage(&self, at: B::Hash, address: Address, index: U256) -> Option<H256> {
 		let tmp: &mut [u8; 32] = &mut [0; 32];
-		index.to_big_endian(tmp);
+		index.write_as_big_endian(tmp);
```

Comment:
Instead of doing this, you can remove all these and update LINE 131:

```patch
-		key.extend(blake2_128_extend(tmp));
+ 		key.extend(blake2_128_extend(&index.to_big_endian()));
```